backend/installfinders/launchers/steam/steam.go

```diff
@@ -99,8 +100,16 @@ func FindInstallationsSteam(steamPath string, launcher string, platform common.L
 
 			var branch common.GameBranch
 			userConfig := manifest["AppState"].(map[string]interface{})["UserConfig"].(map[string]interface{})
-			betakey, ok := userConfig["betakey"]
-			if !ok {
+
+			var betakey string
+			for k, v := range userConfig {
+				if strings.EqualFold(k, "BetaKey") {
+					betakey = v.(string)
+					break
+				}
+			}
+
+			if betakey == "" {
 				branch = common.BranchStable
```

Comment:
```suggestion
			if betakey == "" || betakey == "stable" {
				branch = common.BranchStable
```
I ran the prior code on experimental and it worked, but on stable (Betas -> None in the steam client) it produced this error: `ficsitcli/metadata.go:27 failed to find installations error="unknown beta key public"`, hence this change.